src/main/scripts/org/reaktivity/specification/nukleus/http/control/route/client/controller.rpt

```diff
@@ -33,12 +33,13 @@ write [0x06] "source"
 write 0L
 write [0x06] "target"
 write ${newClientConnectRef}
+write [0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00]
```

Comment:
These `authorization` fields should always be hex literals, right?

Reply:
Yes, I am fixing that in the other spec projects too, will be done in a few mins

Reply:
fixed